src/lib/docker/dockerService.ts

```diff
@@ -347,6 +349,24 @@ class DockerService implements DockerLibrary {
       }
     }
   }
+
+  /**
+   * Renames the docker volume directory on disk when a node is renamed
+   * @param network the network containing the node
+   * @param node the node that's to be renamed
+   * @param newName the new name for the node and directory
+   */
+  async updateDirs(network: Network, node: CommonNode, newName: string) {
+    const volumeDir = (node as AnyNode).implementation
+      .toLocaleLowerCase()
+      .replace('-', '');
+    const oldPath = join(network.path, 'volumes', volumeDir, node.name);
```

Comment:
You can use `const oldPath = nodePath(network, node.implementation, node.name);` to get this path.